apps/desktop-wallet/src/features/transactionsDisplay/transactionDetailsModal/types.ts

```diff
@@ -0,0 +1,25 @@
+/*
+Copyright 2018 - 2024 The Alephium Authors
+This file is part of the alephium project.
+
+The library is free software: you can redistribute it and/or modify
+it under the terms of the GNU Lesser General Public License as published by
+the Free Software Foundation, either version 3 of the License, or
+(at your option) any later version.
+
+The library is distributed in the hope that it will be useful,
+but WITHOUT ANY WARRANTY; without even the implied warranty of
+MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the
+GNU Lesser General Public License for more details.
+
+You should have received a copy of the GNU Lesser General Public License
+along with the library. If not, see <http://www.gnu.org/licenses/>.
+*/
+
+import { AddressHash } from '@alephium/shared'
+import { Transaction } from '@alephium/web3/dist/src/api/api-explorer'
+
+export interface TransactionDetailsModalSectionProps {
```

Comment:
I'm not so sure about having the props separated from the main component 🤔 
Maybe I'm wrong, but I seem to remember you saying that you liked having types close to where they are used.

Reply:
Indeed, but what if we want to share them amongst many components that are in different fileS?

Reply:
Not sure what you mean. The `TransactionDetailsModalSectionProps` is used by 11 components in 11 different files. Where should I put it if not in a dedicated file?

Reply:
NVM, I thought there was a `TransactionDetailsModalSection` component :)